### ISSUE TITLE
Update is not possible when using SSL

### ISSUE THREAD

**Issue by davidbillamboz** [ForestAdmin/lumber#234]
## Expected behavior

`lumber update` should update the app models

## Actual behavior

`lumber update` crashes.

## Failure Logs

```
> Cannot connect to the database due to the following error:
> SequelizeConnectionError: no pg_hba.conf entry for host "89.3.25.84", user "gexvagloeoizgi", database "d91aosrmq4610l", SSL off
```

## Context

* Lumber Package Version: 2.2.3
* Database Dialect: postgres on heroku
* Database Version: xx

I think the relevant part is : 
https://github.com/ForestAdmin/lumber/blob/6191eedc4ed5fa729ea9cbeaa2b161e5ae1ffe28/lumber-update.js#L25-L28

If you compare with the (working) `lumber generate` : 
https://github.com/ForestAdmin/lumber/blob/6191eedc4ed5fa729ea9cbeaa2b161e5ae1ffe28/lumber-generate.js#L29-L34

I think the SSL option is not used.

**Comment by coffenbacher** [comment]
I'm just trying out Lumber and ran into this - it's kind of concerning that such a simple bug has had no activity for a month...

**Comment by arnaudbesnier** [comment]
Hi guys, sorry for the late response.
Let me prioritise investigations on this issue for this week.

**Comment by coffenbacher** [comment]
Sure, no problem. I think it's a really simple one, the suggested fix in the bug report basically worked for me.

**Comment by arnaudbesnier** [comment]
Fixed in the new Lumber version (v2.3.4).
Thanks for your help guys!
